pkg/mark/stdlib/stdlib.go

```diff
@@ -374,6 +374,24 @@ func templates(api *confluence.API) (*template.Template, error) {
 			`</ac:structured-macro>`,
 		),
 
+		/* https://confluence.atlassian.com/conf59/page-tree-macro-792499177.html */
+
+		`ac:tree`: text(
+			`<ac:structured-macro ac:name="pagetree" ac:schema-version="1">`,
+			`<ac:parameter ac:name="root">`,
+			`<ac:link>`,
+			`<ri:page ri:content-title="@self"{{or .Title "" }}/>`,
+			`</ac:link>`,
+			`</ac:parameter>`,
+			`<ac:parameter ac:name="sort">{{ or .Sort "" }}</ac:parameter>`,
+			`<ac:parameter ac:name="excerpt">{{ or .Excerpt "" }}</ac:parameter>`,
```

Comment:
```suggestion
			`<ac:parameter ac:name="excerpt">{{ or .Excerpt "false" }}</ac:parameter>`,
```

Reply:
When I set all the defaults to values, I get this big block of xml in the page that -is not- present when I just insert the macro in the editor.

For this reason I felt it was unecessary.